### PR TITLE
Added ability to update profile information

### PR DESCRIPTION
**Pull request by 74ultra**
- Added update form for profiles
- added update api call in ClientState

### CODE REVIEW

**Review by lydiecherilus — DISMISSED** [review]
Nice work on the update profile functionality!

File: src/components/pages/ClientDashboard/ClientDashboardContainer.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import { useHistory } from 'react-router-dom';
 
 import { Layout, Descriptions, Image } from 'antd';
 
 const ClientDashboardContainer = props => {
   const { name, country, email, avatarUrl } = props.client;
 
+  const history = useHistory();
+
+  console.log(props);
```
Comment (tinomen) [inline src/components/pages/ClientDashboard/ClientDashboardContainer.js]: Lets not let these debug statements find their way into production.

File: src/components/pages/ClientDashboard/RenderClientDashboard.js [hunk 2]
```diff
@@ -6,14 +6,14 @@ import { Spinner } from '../../common';
...
-  const { client } = clientContext;
+  const { client, accountType } = clientContext;
 
   return (
     <div>
       {Object.keys(client).length < 1 ? (
         <Spinner />
       ) : (
-        <ClientDashboardContainer client={client} />
+        <ClientDashboardContainer client={client} type={accountType} />
```
Comment (tinomen) [inline src/components/pages/ClientDashboard/RenderClientDashboard.js]: Take a look at the ExampleList page component. This should be reversed with the ClientDashboardContainer. The render component is where most of the JSX should reside.

File: src/components/pages/LoggedInOptions/GroomerOptions.js [hunk 3]
```diff
@@ -4,7 +4,6 @@ import { Button } from 'antd';
 
 const GroomerOptions = () => {
   let history = useHistory();
-  console.log(history);
```
Comment (tinomen) [inline src/components/pages/LoggedInOptions/GroomerOptions.js:7]: Nice catch. 🎉 Lets get rid of these debug lines.

File: src/components/pages/UpdateProfile/RenderUpdateForm.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import ClientContext from '../../../state/client/clientContext';
+
+const RenderUpdateForm = () => {
+  const clientContext = useContext(ClientContext);
+
+  const { client, accountType, updateProfile } = clientContext;
+
+  return (
+    <div>
+      <UpdateFormContainer
```
Comment (tinomen) [inline src/components/pages/UpdateProfile/RenderUpdateForm.js:12]: Again, the Container should be the parent or main component.

File: src/components/pages/UpdateProfile/UpdateFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    }
+  }, [props]);
+
+  const handleChange = e => {
+    setUpdate({ ...update, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log('submit', update);
```
Comment (tinomen) [inline src/components/pages/UpdateProfile/UpdateFormContainer.js]: debug code

File: src/components/pages/UpdateProfile/index.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export { default as UpdateProfile } from './RenderUpdateForm';
```
Comment (tinomen) [inline src/components/pages/UpdateProfile/index.js:1]: Should be loading the container here

File: src/state/client/ClientState.js [hunk 7]
```diff
@@ -32,6 +32,19 @@ const ClientState = props => {
     });
   };
 
+  // UPDATE CLIENT INFO
+  const updateProfile = async update => {
+    const res = await apiAuth(authState).put(
+      `https://labspt12-express-groomer-c-api.herokuapp.com/profiles`,
```
Comment (tinomen) [inline src/state/client/ClientState.js:38]: Can we replace the URI with the env var REACT_APP_API_URI?

Something like `'${process.env.REACT_APP_API_URI}/profile'`

**Review by tinomen — CHANGES_REQUESTED** [review]
It looks good over all; just a few things that should be fixed.

**Pull request closed without merging**